src/pages/favorites/favorites.component.spec.ts

```diff
@@ -71,4 +71,37 @@ describe('Favorites Component', () => {
 
     });
   });
+  describe('removeRoute', () => {
+    // Make an arbitrary route object available to each test
+    beforeAll(() => {
+      this.route = {
+        RouteId: 20035,
+        GoogleDescription: 'Campus Shuttle Southbound',
+        ShortName: '35',
+        RouteAbbreviation: '35',
+        Color: '00467E'
+      }
+    });
+    it('removes an item from the routes array', () => {
+      // Set the class-wide this.routes array in the component
+      component.routes = [ this.route ];
```

Comment:
👍 scope is difficult to understand in the tests - in general:
- Assign `this.xyz` variables in `beforeEach` or `beforeAll` and access them in `it()` blocks.  These variables have nothing to do with the `this.zyx` variables in the component.
- Set and get `this.zyx` variables that normally reside in the component via `component.zyx` in the `it()` blocks.